### ISSUE TITLE
Inflating error in API 19

### ISSUE THREAD

**Issue by Andur1l** [timrijckaert/FloatingMusicActionButton#2]
Works without problem in Nougat (API 25), but gets a:

android.view.InflateException: Binary XML file line #156: Error inflating class be.rijckaert.tim.animatedvector.FloatingMusicActionButton

in KitKat (API 19).

**Comment by jdsingh** [comment]
Same here. Getting crash reports from Android 4.2.2

**Comment by rpattabi** [comment]
I haven't tried this library yet, but for this problem I believe support library's vector drawable support shall be enabled.

Step 1: In app -> build.gradle

```
defaultConfig {
  vectorDrawables.useSupportLibrary = true
}
```

Step 2: In your Application class or Activity static initializer

```
AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
```

**Comment by jdsingh** [comment]
@rpattabi Yeah, that's what I did to fix this problem. But I'm no longer using this library.
